tests/wpunit/CommentObjectQueriesTest.php

```diff
@@ -664,61 +672,76 @@ public function testQueryingAvatarOnUserAuthorsIsValidForPublicAndAuthenticatedR
 
 		// create a comment with a guest author as the author
 		$comment_by_comment_author_id = $this->createCommentObject([
-			'user_id' => 0
+			'user_id'              => 0,
+			'comment_author_email' => 'guest@email.test',
 		]);
 
 		$comment_by_user_id = $this->createCommentObject();
 
 		$query = '
 		query GetCommentAuthorWithAvatar($id:ID!){
-		  comment( id: $id ) {
-		    author {
-              node {
-                __typename
-                url
-                name
-                avatar {
-                  url
-                }
-              }
-            }
-		  }
+			comment( id: $id ) {
+				author {
+					node {
+						__typename
+						url
+						name
+						avatar {
+							url
+						}
+					}
+				}
+			}
 		}
 		';
 
-		$global_id = \GraphQLRelay\Relay::toGlobalId( 'comment', $comment_by_comment_author_id );
+		$variables = [
+			'id' => \GraphQLRelay\Relay::toGlobalId( 'comment', $comment_by_comment_author_id ),
+		];
 
-		$actual = graphql( [
-			'query'     => $query,
-			'variables' => [
-				'id' => $global_id,
-			],
-		] );
+		$actual = $this->graphql( compact( 'query', 'variables' ) );
 		$this->assertArrayNotHasKey( 'errors', $actual );
-		codecept_debug( $actual );
 
 		$typename = $actual['data']['comment']['author']['node']['__typename'];
 
 		$this->assertSame( 'CommentAuthor', $typename );
 		$this->assertNotEmpty( $actual['data']['comment']['author']['node']['avatar']['url'] );
 
-		$global_id = \GraphQLRelay\Relay::toGlobalId( 'comment', $comment_by_user_id );
+		// Ensure avatar is the same when logged in.
```

Comment:
I forgot to save this as a separate commit. 👆 is the new test

Reply:
@justlevine thanks for pointing this out. Makes reviewing much easier 🙏🏻